flavors/posture.go

```diff
@@ -122,7 +121,7 @@ func (bt *posture) Run(b *beat.Beat) error {
 		return err
 	}
 
-	bt.dataStop = bt.data.Run(bt.ctx)
+	bt.data.Run(bt.ctx)
```

Comment:
`Run` knows it needs to stop once `bt.ctx` is closed. didn't find a need to add a Stop function